apps/barista-design-system/src/components/icon-color-wheel/icon-color-wheel.ts

```diff
@@ -154,6 +163,14 @@ export class BaIconColorWheel {
     this._iconId = `ba-icon-color-wheel-iconId-${++uniqueColorwheelId}`;
   }
 
+  ngAfterViewInit(): void {
+    this._renderer2.setProperty(
```

Comment:
can you please do it without the renderer since it should be avoided.

```typescript
this._symbol.nativeElement.innerHTML = this._svg
```

Reply:
Will this work in SSR then, when using nativeElement?